examples/next_pykiso2/pykiso_as_simulator/serial_simulation.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+
+import pykiso
+
+pykiso.load_config(Path(__file__).parent.resolve() / "serial.yaml")
+
+
+from pykiso.lib.auxiliaries.communication_auxiliary import CommunicationAuxiliary
+
+sender = CommunicationAuxiliary.get_instance('com_aux_sender')
```

Comment:
Cool would be a factory.
I mean the info that the aux is of type  `CommunicationAuxiliary` is already in the yaml.

kiso_conf = pykiso.load_config(Path(__file__).parent.resolve() / "serial.yaml")
kiso_conf.get_instance('com_aux_sender')

maybe for the future ?!

Reply:
One step after the other ^^

Reply:
I'm not sure on how to resolve the the typing resolution for that